lib/context-util.c

```diff
@@ -139,18 +139,18 @@ tcti_tabrmd_init (void)
 
     rc = tss2_tcti_tabrmd_init(NULL, &size);
     if (rc != TSS2_RC_SUCCESS) {
-        LOG_ERR ("Failed to get size for TABRMD TCTI context: 0x%" PRIx32, rc);
+        LOG_ERR ("Failed to get size for TABRMD TCTI context: 0x%" PRIx32 "\n", rc);
```

Comment:
Anything warning or debug wise should go to stderr. Anything the tool *should* output, should go to stdout.